examples/undo-tree/server.js

```diff
@@ -1,6 +1,6 @@
 /**
  * This is the main server entry point into the
- * undo-tree example.
+ * drawing example.
```

Comment:
Renamed to painter!

On Tue, Jan 26, 2016 at 11:05 AM, Lawson Kurtz <notifications@github.com>
wrote:

> In examples/undo-tree/server.js
> <https://github.com/vigetlabs/microcosm/pull/94#discussion_r50855268>:
>
> > @@ -1,6 +1,6 @@
> >  /**
> >   * This is the main server entry point into the
> > - * undo-tree example.
> > + * drawing example.
>
> but the directory name is still undo-tree :-(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vigetlabs/microcosm/pull/94/files#r50855268>.
>